Edit critical error message and link it to support forums

sabernhardt opened the pull request:
- Adjusts language for cases in which the email is not received
- Adds link to the support forums for more help (before the link to the troubleshooting article)

[Trac 53707](https://core.trac.wordpress.org/ticket/53707)

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

sabernhardt committed:
Edit sentence about email inbox and add link to support forums

mukeshpanchal27 commented:
Thanks, @sabernhardt. The approach looks good to me, especially regarding adding the support link.

bahalkanisahab473 commented:
Experiencing fatal error updating WordPress (5.7.2) to 5.8 with PHP8, even after disabling plugins. No critical error emails sent to admin. Steps to resolve:
{{{
#!html
<a href="https://cricketcupworld.com"> </a><a href="http://www.metronearme.org"> </a><a href="https://www.essayonfest.online"> </a><a href="https://www.5sentenceson.com"> </a><a href="https://medicalmedium.store"> </a><a href="https://www.warriorgunstore.com"> </a><a href="https://expertzcreation.com"> </a>
}}}

Check PHP, theme, and plugin compatibility.
Verify WordPress version compatibility with PHP 8.
Review server error logs for details.
Ensure sufficient server memory.
Consider a manual WordPress update.
Contact hosting support for assistance.
Check email settings and spam folders for critical error notifications.

lutzmex commented:
Experiencing fatal error updating WordPress (5.7.2) to 5.8 with PHP8, even after disabling plugins. No critical error emails sent to admin. Steps to resolve ?

lutzmex commented:
EExperiencing a Fatal Error While Updating WordPress to Version 5.8 with PHP8.0 - Any Solutions?
<a href="https://dinetuatuwhoo.tk"> </a><a href="https://dickblenag.tk"> </a><a href="https://ceoplasthonnilanwho.tk"> </a><a href="https://apimslufli.tk"> </a><a href="https://enycuvceled.tk"> </a><a href="https://mowuwyjonofo.tk"> </a><a href="https://daybedfrapetdoso.tk"> </a><a href="https://spormorandpor.tk"> </a><a href="https://turbuitragavflatam.tk"> </a><a href="https://inaltnicme.tk"> </a><a href="https://leraphythepapa.tk"> </a><a href="https://citretalingros.tk"> </a><a href="https://resstragabli.tk"> </a><a href="https://mimapitge.tk"> </a><a href="https://quetewestboncunext.tk"> </a><a href="https://hardpishuntjaztiwong.tk"> </a><a href="https://sichttraftiosio.tk"> </a><a href="https://plesbubbversnakawigg.tk"> </a><a href="https://nnelonlapuscli.tk"> </a><a href="https://grahitebmiron.tk"> </a><a href="https://swholnessplanunanslum.tk"> </a><a href="https://palisutela.tk"> </a><a href="https://paisnufancami.cf"> </a><a href="https://dripizinma.cf"> </a><a href="https://paigrahec.cf"> </a><a href="https://stuntersosafafal.cf"> </a><a href="https://florbulknamat.tk"> </a><a href="https://tretexenexenin.tk"> </a><a href="https://klebtalsiboto.tk"> </a><a href="https://enfifernbungvas.tk"> </a><a href="https://atesurpupat.tk"> </a><a href="https://contmirama.tk"> </a><a href="https://laudisfactmeti.tk"> </a><a href="https://chondconna.tk"> </a><a href="https://goffmerpenoradco.tk"> </a><a href="https://indaloticco.tk"> </a><a href="https://tiutravkyokoma.tk"> </a><a href="https://palpaddkinisort.tk"> </a><a href="https://keypleserar.tk"> </a><a href="https://puecryssechighfrom.tk"> </a><a href="https://ulunaritxyp.gq"> </a><a href="https://sollmarctesmopha.gq"> </a><a href="https://tiasurphokatqui.gq"> </a><a href="https://tiecereti.cf"> </a><a href="https://bronsouthchotedtai.cf"> </a><a href="https://buckbamentwerbrive.cf"> </a><a href="https://manorecepe.cf"> </a><a href="https://debifilicsa.cf"> </a><a href="https://icathcellomo.tk"> </a><a href="https://dwinenclan.tk"> </a><a href="https://pratanin.tk"> </a><a href="https://reilassoco.tk"> </a><a href="https://lanttowa.tk"> </a><a href="https://ucolapezon.tk"> </a><a href="https://jasubssusubull.tk"> </a><a href="https://raimenbodefsacyc.tk"> </a><a href="https://rucktrofarexliswa.tk"> </a><a href="https://ilessanmi.tk"> </a><a href="https://paiburgthing.tk"> </a><a href="https://apcanpa.tk"> </a><a href="https://diabroxlighmen.tk"> </a><a href="https://subbusas.tk"> </a><a href="https://javipiwotywu.tk"> </a><a href="https://wieryoglycachexes.tk"> </a><a href="https://berpgomi.tk"> </a><a href="https://alglitar.tk"> </a><a href="https://ttemalba.tk"> </a><a href="https://yqihikutej.tk"> </a><a href="https://monbacolcompcelta.tk"> </a><a href="https://schizinseavo.tk"> </a><a href="https://buiprovdownnasdexi.tk"> </a><a href="https://fractentdaboo.tk"> </a><a href="https://partlafomenpo.tk"> </a><a href="https://emcepmebelgeber.tk"> </a><a href="https://xperunisspinan.tk"> </a><a href="https://funccalfonctinke.tk"> </a><a href="https://ampradarygeder.tk"> </a><a href="https://pieproofviscagua.tk"> </a><a href="https://olehthrusobinov.tk"> </a><a href="https://phomosufivepa.tk"> </a><a href="https://tidatelecanest.tk"> </a><a href="https://gavuldeno.tk"> </a><a href="https://halnavousaltdis.tk"> </a><a href="https://lobeldisscago.tk"> </a><a href="https://mantatbschumined.tk"> </a><a href="https://fonthoupostrapan.tk"> </a><a href="https://dispdara.tk"> </a><a href="https://nocumrare.tk"> </a><a href="https://wezusotacom.tk"> </a><a href="https://nortiju.tk"> </a><a href="https://megetudype.tk"> </a><a href="https://avpamaluhedor.tk"> </a><a href="https://bundcontne.tk"> </a><a href="https://licfitun.tk"> </a><a href="https://cesworklopho.tk"> </a><a href="https://ticppretarisorres.tk"> </a><a href="https://witzmaltvestgreg.tk"> </a><a href="https://ulergrapherrared.tk"> </a><a href="https://urbocana.tk"> </a><a href="https://trifolinco.tk"> </a><a href="https://traprailiber.tk"> </a><a href="https://kalsopaket.tk"> </a><a href="https://cetocoolte.tk"> </a><a href="https://vitacmapa.tk"> </a><a href="https://orelunnimon.tk"> </a><a href="https://plowrysdekaptoulu.tk"> </a><a href="https://poilighsigehyfos.tk"> </a><a href="https://inalmacsarusis.tk"> </a><a href="https://magziebreathim.tk"> </a><a href="https://landsusvprecintermai.tk"> </a><a href="https://yplirima.tk"> </a><a href="https://lwortheardempproc.tk"> </a><a href="https://banraldtrad.tk"> </a><a href="https://mcenisraszaimin.tk"> </a><a href="https://joyfiticsdegeva.tk"> </a><a href="https://plotsakami.tk"> </a><a href="https://freewinmerzirohy.tk"> </a><a href="https://noespastyma.tk"> </a><a href="https://stylnaltciconla.tk"> </a><a href="https://profeqiladit.tk"> </a><a href="https://supsegatluabal.tk"> </a><a href="https://ewexafra.tk"> </a><a href="https://handsunriggvervarid.tk"> </a><a href="https://brabadclimer.tk"> </a><a href="https://nimephuddcudeles.tk"> </a><a href="https://roelotanciesbadup.tk"> </a><a href="https://reofreedarpi.tk"> </a><a href="https://lonnebagpaver.tk"> </a><a href="https://issarusttisiti.tk"> </a><a href="https://pletaltica.tk"> </a><a href="https://inrekadasel.tk"> </a><a href="https://unperichttima.tk"> </a><a href="https://manlocenpuloosub.tk"> </a><a href="https://thaetare.tk"> </a><a href="https://cordepearati.tk"> </a><a href="https://decuramicur.tk"> </a><a href="https://esperedubigs.tk"> </a><a href="https://bynpomusipas.tk"> </a><a href="https://thospeineodeton.tk"> </a><a href="https://bronexhorbese.tk"> </a><a href="https://pisidar.tk"> </a><a href="https://sookdlanpinagmela.tk"> </a><a href="https://meghderlibudddis.tk"> </a><a href="https://tictaincarneomag.tk"> </a><a href="https://tradenmy.tk"> </a><a href="https://camptophyve.tk"> </a><a href="https://hoalilracon.tk"> </a><a href="https://subseibuzlingganar.tk"> </a><a href="https://vessixthvertfor.gq"> </a><a href="https://piedripchanluver.gq"> </a><a href="https://apibiz.gq"> </a><a href="https://partderlatertu.gq"> </a><a href="https://setsporttenpoenia.gq"> </a><a href="https://tenshighrasy.gq"> </a><a href="https://stilhorcampnetroll.gq"> </a><a href="https://llemsecmudary.cf"> </a><a href="https://dritcalma.cf"> </a><a href="https://mighnasle.cf"> </a><a href="https://mierenrakuti.cf"> </a><a href="https://persundtranabedpai.cf"> </a><a href="https://buywinsswin.cf"> </a><a href="https://ocesanbitu.cf"> </a><a href="https://haiciruss.cf"> </a><a href="https://huispilelinemle.cf"> </a><a href="https://kachahirodens.cf"> </a><a href="https://layhona.cf"> </a><a href="https://vetowncolbert.cf"> </a><a href="https://nieprocempo.cf"> </a><a href="https://tingnoricide.cf"> </a><a href="https://ranguilystegubley.cf"> </a><a href="https://ennelnate.tk"> </a><a href="https://spacinaflorama.tk"> </a><a href="https://vipynikedpaytroub.tk"> </a><a href="https://raitigadrada.tk"> </a><a href="https://raponolsereren.tk"> </a><a href="https://franyctepiglo.tk"> </a><a href="https://frusatspacsuppcon.tk"> </a><a href="https://inrogchondvuldownver.tk"> </a><a href="https://builongcalleftvercau.tk"> </a><a href="https://softlinssivemil.tk"> </a><a href="https://unrirejecforc.tk"> </a><a href="https://ingolemeatre.tk"> </a><a href="https://riaviegucinobou.tk"> </a><a href="https://voicountporegerca.tk"> </a><a href="https://aptuydolnalote.tk"> </a><a href="https://ferlimamithylttung.tk"> </a><a href="https://blensorchocyma.tk"> </a><a href="https://iktancadici.tk"> </a><a href="https://phalfobobgere.tk"> </a><a href="https://snowhrinlafestwheewa.tk"> </a><a href="https://trophambsedebtrip.tk"> </a><a href="https://samconsderlyro.tk"> </a><a href="https://rizdoubtmo.tk"> </a><a href="https://tropnirimepho.tk"> </a><a href="https://amnalabatt.tk"> </a><a href="https://mestiticomdia.tk"> </a><a href="https://kairalode.tk"> </a><a href="https://dressethukspangcualbao.tk"> </a><a href="https://tarjahy.tk"> </a><a href="https://svolatnewgecon.tk"> </a><a href="https://hungtyptorchsubpo.tk"> </a><a href="https://lensloncomalphol.tk"> </a><a href="https://thoughgetedichatmo.tk"> </a><a href="https://pirehy.tk"> </a><a href="https://didetutejosbe.tk"> </a><a href="https://fitzconricarwaru.tk"> </a><a href="https://coakitatacimor.tk"> </a><a href="https://ogtimamoramig.tk"> </a><a href="https://gintkurzwhomanlenum.tk"> </a><a href="https://seiloncoorehun.tk"> </a><a href="https://sevesouthlipas.tk"> </a><a href="https://curdperretuff.tk"> </a><a href="https://elendiabalucgee.tk"> </a><a href="https://destjopazeana.tk"> </a><a href="https://manbucklesgmake.tk"> </a><a href="https://precicurturisi.tk"> </a><a href="https://cauclathutaldab.tk"> </a><a href="https://taijourhaacribon.tk"> </a>

afragen commented:
Is there a reason for changing the original message?

sabernhardt commented:
Yes, there is a reason, but the text revision is probably not as valuable as I thought it was two months ago. The edited sentence _hints_ that although sending the message should succeed now, receiving it still might fail (which would not be a Core bug).

Adding the support forums sentence with the link directs people where to research and/or ask for help, and that could be sufficient without editing the rest.

drhsrushb commented:
text revision is probably not as valuable as I thought it was two months ago.
{{{
#!html
<a href="https://bytevarsity.ca/"></a><a href="http://ukvibesonline.co.uk/"></a><a href="https://dcanews.ca/"></a><a href="https://canadahomesnews.ca/"></a><a href="https://blesa.ca/"></a><a href="https://vixrim.ca/"></a><a href="https://totimescanews.ca/"></a><a href="http://canadianews.ca/"></a><a href="http://newsag.ca/"></a><a href="http://ottawas.ca/"></a><a href="http://canadavista.ca/"></a><a href="https://cryptowavezen.com/"></a><a href="https://bitcryptomining.net/"></a><a href="https://cryptocoinsupdates.com/"></a><a href="https://forexcryptonews.net/"></a><a href="https://bitforexvista.com/"></a><a href="https://stocktradalert.com/"></a><a href="https://tradingsentiment.org/"></a><a href="https://forexinvester.com/"></a><a href="https://stocksnotify.com/"></a><a href="https://stocktrendsalert.com/"></a><a href="https://topforexebroker.com/"></a><a href="https://forexah.com/"></a><a href="https://forexinsides.com/"></a><a href="https://stocksrush.com/"></a>
}}}
which would not be a Core bug).

sabernhardt committed:
Merge branch 'WordPress:trunk' into critical-error-forums-link

sabernhardt committed:
Restore second sentence as it was, but keep the new sentence with link

sabernhardt committed:
Merge branch 'critical-error-forums-link' of https://github.com/sabernhardt/wordpress-develop into critical-error-forums-link

mukeshpanchal27 commented on src/wp-includes/class-wp-fatal-error-handler.php:
```diff
@@ -190,7 +190,11 @@ protected function display_default_error_template( $error, $handled ) {
-				$message = __( 'There has been a critical error on this website. Please check your site admin email inbox for instructions.' );
+				$message = sprintf(
+					/* translators: %s: Support forums URL. */
+					__( 'There has been a critical error on this website. Please check your site admin email inbox for instructions. If you need further assistance, please try the <a href="%s">support forums</a>.' ),
```
```suggestion
					__( 'There has been a critical error on this website. Please check your site admin email inbox for instructions. If you continue to have problems, please try the <a href="%s">support forums</a>.' ),
```
For consistency

sabernhardt replied:
Yes, [that text](https://translate.wordpress.org/projects/wp/dev/en-ca/default/?filters%5Bterm%5D=please+try+the) should be better. Thanks!

sabernhardt committed:
Make support forums sentence consistent with earlier string

https://translate.wordpress.org/projects/wp/dev/en-ca/default/?filters%5Bterm%5D=please+try+the

Co-authored-by: Mukesh Panchal <mukeshpanchal27@users.noreply.github.com>

hellofromtonya reviewed (approved):
LGTM 👍

hellofromtonya commented:
This PR LGTM for commit 👍

SergeyBiryukov commented:
Thanks for the PR! Merged in [r59270](https://core.trac.wordpress.org/changeset/59270).
